book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] An "add_rental" method should be added as the last code reviewer mentioned. You can implement this as you did in the person class.

Reply:
Thank you so much for pointing this out, it was a slight oversight and I have fixed the requested change. However, the interface component you talked about is required in the `person.rb` file. 

This is where the decorator pattern was implemented and person class inherits from it, I believe it's not redundant.

Thank you once again for the thorough review 🏅